htdocs/bookmarks/bookmarks.lib.php

```diff
@@ -36,28 +36,39 @@ function printDropdownBookmarksList()
 
 	$langs->load("bookmarks");
 
+	$authorized_var=array('page','limit','optioncss','contextpage');
```

Comment:
ok to add the param limit, optioncss and contextpage as param to include into the bookmark url to match a post.
However, i don't think "page" is a good idea. When changing data, the content of page is changing too. When we add the result of a post in bookmarks it means we want the result of some filters. It looks strange to me to say "i want to bookmark result when searching on abc but i want to go by default on the page 2 of the result".
Don't you think so ?

Reply:
You'r right in Dolibarr Core this param are never set in post, but maybe so external module can play with it. No Use case for now, just a guess